src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -65,4 +71,9 @@ private function getExpressionLanguage()
 
         return $this->expressionLanguage;
     }
+
+    private function getPropertyAccessor()
+    {
+        return PropertyAccess::createPropertyAccessor();
```

Comment:
the exception in src/Symfony/Component/Validator/Constraints/Expression.php should be moved here IMHO:
since the property is public, the check in the constructor is very fragile, only here would be robust

Reply:
I would rather make the property `private` then. Catching these mistakes when the constraint is created allows to spot issues earlier which improves DX.

Reply:
I made the property `private` and added a magic `__get` like it was done in the `File` Constraint.